projects/cdk/services/resize.service.ts

```diff
@@ -28,25 +26,26 @@ export class TuiResizeService extends ResizeObserverService {
         @Self() @Inject(TuiDestroyService) destroy$: Observable<void>,
         @Inject(RESIZE_OBSERVER_SUPPORT) support: boolean,
         @Inject(RESIZE_OPTION_BOX) box: ResizeObserverBoxOptions,
-        @Inject(ANIMATION_FRAME) animationFrame$: Observable<number>,
     ) {
         super(elementRef, ngZone, support, box);
 
         return this.pipe(
-            catchError(() =>
```

Comment:
`catchError` with a fallback to `requestAnimationFrame` is only here for browsers that do not support `ResizeObserver`. It is necessary to keep that part.

Reply:
<img width="1373" alt="image" src="https://user-images.githubusercontent.com/12021443/207065558-3203ab8d-4f2a-4aaa-a336-e7778d2847ba.png">

do we need support old browsers?

Reply:
fixed

Reply:
Yes, we moved support up to Safari 12.1 for `IntersectionObserver` in 3.0.0, but `ResizeObserver` requires Safari 13+.